azerothcore/mod-cfbg
# feat(random-race): randomization of races and related morph based on gender

### Changes proposed
Randomize the races of the player during the change faction.

### HOW TO TEST
- join battleground with 2 players of the same faction
- check if the second one gets a random race which respects his class
- leave battleground and join again to see if the player get a different race

Note: if you are a human warrior and join battleground as horde, you can't be Blood Elf because a BE can't be warrior, but probably you will be randomized for all other races.

**Hint**: from worldserver.conf remove the deserter buff
